i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/chats.mdx

```diff
@@ -194,6 +194,18 @@ Das Laden des Archivs kann einige Zeit in Anspruch nehmen.
   </TabItem>
 </Tabs>
 
+## Den Online-Status anderer Nutzer prüfen
+
+Sie können den Online-Status anderer Nutzer auf den Avataren in der Raumliste, der Kontaktliste (beim Erstellen eines neuen Chats), in der Titelleiste von 1:1 Chats sowie in Gruppenchats in den einzelnen Nachrichten und der Mitgliederliste sehen.  
+
+Welches Symbol zeigt welchen Online-Status?
+
+<b>Online</b> &rarr; Grüner Punkt auf dem Avatar
```

Comment:
Please check this, I believe in the help center we do not use the term Avatar anywhere. It is the profile picture or the initials.